package-lock.json

```diff
@@ -46,6 +46,7 @@
         "keycode": "^2.2.0",
         "lodash": "^4.17.20",
         "lodash-es": "^4.17.14",
+        "moment": "^2.24.0",
```

Comment:
also from lock file 🚔  (lock file is automatically generated after `npm install`)